### DISCUSSION TITLE
rAudio 20240707: Spurious library view as filesystem path

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#1708] [Fixed Issues]
Selecting a sub-folder in the library view presents the contents as filesystem path:
<img src="https://github.com/rern/rAudio/assets/25383513/ca330280-e83d-4f44-b1e0-a3ea8a5673d8" width=600>

After going back and forth the correct contents are displayed:
<img src="https://github.com/rern/rAudio/assets/25383513/ec0d6d3e-de92-46ad-a95a-8e4334f3688b" width=600>

**Comment by klassikmann** [comment]
Nobody faces the same bug?

**Reply by clriis** [reply]
Yes. Same here...

**Reply by R101G** [reply]
And here, but not every time.

**Reply by rern** [reply]
Please try the UPDATE.

**Reply by klassikmann** [reply]
@rern 
Issue seems to be solved on devices capable of swiping (smartphones, teblets).
**BUT**
Using desktop browsers (Windows, Linux) without swiping support there is no way to switch to the library or playback view by clicking.
Even restoring the last backup doesn't remedy the navigating behaviour.

**EDIT**:
The issue after the **BUT** seems to be a non-issue provoked by the restore of the last backup.

**Reply by klassikmann** [reply]
@rern 
Additional issue with UPDATE on desktop browsers without swiping support: Help guide bitmaps doesn't scale anymore.

**Reply by klassikmann** [reply]
@rern 
Yet another topic with a potential relationship to the issues on desktop browsers without swiping support:
The last release notes contain a line
`Add: Settings - Keyboard navigation`
Where can I find such a setting?

**Reply by ceejayemm** [reply]
@klassikmann 
In the main Guide (Settings / Addons / ?) keyboard navigation options are shown in pale blue:

![image](https://github.com/user-attachments/assets/dcfbd8ba-a8b8-4a70-8be3-f5371c51ff3f)

eg KB: Home and KB: Backspace

I think !

Chris

**Comment by klassikmann** [comment]
@ceejayemm 
Hi Chris,

thank you for the clue.

Herbert

**Comment by klassikmann** [comment]
Issue described in the 1st post is fixed in 20240719.
